runtime/karura/src/lib.rs

```diff
@@ -132,6 +137,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
 	transaction_version: 1,
+	state_version: 1,
```

Comment:
Just a heads-up: basti said on element that this has to be set to 0. I had it set to 1 initially as well, but it caused us to fail to produce blocks on a rococo-local setup